Kitodo/src/main/java/de/sub/goobi/forms/GranularityForm.java

```diff
@@ -262,7 +262,11 @@ public Long getNumberOfPages() {
      * @return the total number of pages of the digitization project
      */
     public Long getNumberOfPagesPerIssue() {
-        return numberOfPages != null ? numberOfPages / course.countIndividualIssues() : null;
+        if (course.countIndividualIssues() != 0) {
+            return numberOfPages != null ? numberOfPages / course.countIndividualIssues() : null;
+        } else {
+            return null;
```

Comment:
We would never return `null` as this value cause null pointer exceptions or add a lot of null value checks.

Reply:
so what should be return here? 0?

Reply:
Yes, this should be ok and correct value.